src/model_audio_to_video.py

```diff
@@ -0,0 +1,188 @@
+import torch
+import torch.nn as nn
+import pandas as pd
+from torch.utils.data import Dataset
+import librosa
+from preprocess.adio_to_image.preprocess import preprocess_audio
+from torch.utils.data import random_split
+from torch.utils.data import DataLoader
+import torch.optim as optim
+import numpy as np
+import warnings
+
+warnings.filterwarnings("ignore")
+
+
+def load_metadata(file_path):
+    df = pd.read_csv(file_path)
+    num_classes = df['primary_label'].nunique()
+    return df, num_classes
+
+
+class CustomCNN(nn.Module):
```

Comment:
Just an idea: The whole model definition could also be seperated into a different audio_to_image_model_def.py file.

Reply:
Good Idea. I also made it to its own module